src/ontology/edits/catalog-v001.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <catalog prefer="public" xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
+    <uri id="Imports Wizard Entry" name="http://purl.obolibrary.org/obo/iao/2019-06-07/iao.owl" uri="../imports/iao-annotation-module.owl"/>
```

Comment:
Because this is a module that we create just for the annotations, I think we should change the URI of the module just as we have for the others. So the URI could be: http://open-energy-ontology.org/ontology/imports/iao-annotation-module.owl